minijinja/src/filters.rs

```diff
@@ -641,13 +641,33 @@ mod builtins {
     }
 
     /// Returns the sorted version of the given list.
+    /// ```jinja
+    /// {{ [1, 3, 2, 4]|sort }} -> [4, 3, 2, 1]
+    /// {{ [1, 3, 2, 4]|sort(reverse=true) }} -> [1, 2, 3, 4]
+    /// # Sort users by age attribute in descending order.
+    /// {{ users|sortattr(attribute="age") }}
+    /// # Sort users by age attribute in ascending order.
+    /// {{ users|sortattr(attribute="age", reverse=true) }}
+    /// ```
     #[cfg_attr(docsrs, doc(cfg(feature = "builtins")))]
-    pub fn sort(state: &State, value: Value, reverse: Option<bool>) -> Result<Value, Error> {
+    pub fn sort(
```

Comment:
For keyword arguments you have to use something `kwargs: Value` and then check `kwargs.is_kwargs()` in which case it's an object holding keyword arguments. There is currently no good API for this sadly. You can look at the `map` filter for how it's used there.